api/users.go

```diff
@@ -18,7 +18,7 @@ const (
 	executeActionsEmailPath    = userIDPath + "/execute-actions-email"
 	sendReminderEmailPath      = "/auth/realms/:realm/onboarding/sendReminderEmail"
 	smsAPI                     = "/auth/realms/:realm/smsApi"
-	sendNewEnrolmentCode       = smsAPI + "/sendNewCode"
+	sendSmsCode                = smsAPI + "/sendNewCode"
```

Comment:
sendNewCode... which type of code ? we could rename this sendSmsCode ? (and more precisely, which kind of sms code)
(would require a change in keycloak-sms too)

Reply:
in the URL there is already /smsApi/ so it would be redundant.